src/bootstrap/configure.py

```diff
@@ -21,45 +21,47 @@ def __init__(self, name, rustbuild, desc, value):
 
 
 options = []
+option_defaults = {}
 
 
-def o(*args):
+def o(*args, default):
+    option_defaults[args[0]] = default
     options.append(Option(*args, value=False))
 
 
 def v(*args):
     options.append(Option(*args, value=True))
 
 
-o("debug", "rust.debug", "enables debugging environment; does not affect optimization of bootstrapped code")
-o("docs", "build.docs", "build standard library documentation")
-o("compiler-docs", "build.compiler-docs", "build compiler documentation")
-o("optimize-tests", "rust.optimize-tests", "build tests with optimizations")
-o("verbose-tests", "rust.verbose-tests", "enable verbose output when running tests")
-o("ccache", "llvm.ccache", "invoke gcc/clang via ccache to reuse object files between builds")
-o("sccache", None, "invoke gcc/clang via sccache to reuse object files between builds")
-o("local-rust", None, "use an installed rustc rather than downloading a snapshot")
+o("debug", "rust.debug", "enables debugging environment; does not affect optimization of bootstrapped code", default=False)
+o("docs", "build.docs", "build standard library documentation", default=True)
+o("compiler-docs", "build.compiler-docs", "build compiler documentation", default=False)
```

Comment:
If this prints `--disable-docs`, then the current description afterwards must be negated.  E.g. to "skip building compiler documentation".